### ISSUE TITLE
Itermplot on R

### ISSUE THREAD

**Issue by gzagatti** [daleroberts/itermplot#25]
That's a very cool backend! I was wondering if you have had any chance of using it for plotting R plots from within R. 

I'm sure python and R have completely different architectures, but I thought it was worth asking.

**Comment by daleroberts** [comment]
No, unfortunately I haven't got around to porting it to R yet. Glad you
like the project though :)
